__tests__/app.test.js

```diff
@@ -3,33 +3,45 @@ const app = require("../db/app.js");
 const db = require("../db/connection.js");
 const testData = require("../db/data/test-data");
 const seed = require("../db/seeds/seed.js");
-
+const endpoints = require("../endpoints.json");
 
 beforeEach(() => seed(testData));
 afterAll(() => db.end());
 
 describe("CORE: GET /api/topics", () => {
-  test("should get an array of topic objects, each of which should have slug and description", () => {
+  test("GET:200 should get an array of topic objects, each of which should have slug and description", () => {
     return request(app)
       .get("/api/topics")
       .expect(200)
       .then((response) => {
         const { topics } = response.body;
         expect(topics.length).toBe(3);
         topics.forEach((topic) => {
-            expect(topic).toMatchObject({
-              description: expect.any(String),
-              slug: expect.any(String)
-            });
+          expect(topic).toMatchObject({
+            description: expect.any(String),
+            slug: expect.any(String),
           });
+        });
       });
-  })
+  });
   test("GET:404 responds with an appropriate status and error message when provided a non existent endpoint", () => {
     return request(app)
-    .get("/api/nonexistentendpoint")
-    .expect(404)
-    .then((response) => {
-      expect(response.body.msg).toBe('Not found');
-    })
+      .get("/api/nonexistentendpoint")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Not found");
+      });
+  });
+});
+
+describe("CORE: GET /api", () => {
+  test("GET: 200 should get a json object describing all available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const { endpoints } = response.body;
+        expect(endpoints).toBe(endpoints);
```

Comment:
This is a false pass as you are comparing the result to itself. 
instead you either need to rename the variable` const endpoints = require("../endpoints.json");`
or do `expect(endpoints).toEqual(response.body)`

Due to this line  ` const { endpoints } = response.body` it is causing confusion